ocaml/file_formats/cms_format.ml

```diff
@@ -26,8 +26,10 @@ type cms_infos = {
   cms_comments : (string * Location.t) list;
   cms_sourcefile : string option;
   cms_builddir : string;
+  cms_loadpath : Load_path.paths;
   cms_source_digest : Digest.t option;
-  cms_uid_to_loc : Location.t Shape.Uid.Tbl.t;
+  cms_initial_env : Env.t;
+  cms_uid_to_loc : string Location.loc Shape.Uid.Tbl.t;
```

Comment:
As discussed, I think it would save space to store a `(Uid.t * string Location.loc) array` (or similar), instead of serializing the table directly. Perhaps that improvement goes in a separate PR, though.

Reply:
This was an easy enough change that I just went ahead and did it.